docs/openapi/openapi.swagger.yaml

```diff
@@ -57280,7 +57280,7 @@ definitions:
     properties:
       receiver:
         type: string
-      receiver_chainId:
+      receiver_chain_id:
```

Comment:
I'm pretty sure this will break existing indexers.

Reply:
I've deleted this from the changed files